docs/Release_Guide/metviewer_official.rst

```diff
@@ -8,4 +8,14 @@ Create a new vX.Y.Z official release from the develop branch.
 .. include:: release_steps/open_release_issue.rst
 .. include:: release_steps/clone_project_repository.rst
 .. include:: release_steps/checkout_develop_branch.rst
-.. include:: release_steps/create_release_feature_branch.rst
+.. include:: release_steps/metviewer/update_version_official.rst
+.. include:: release_steps/update_release_notes_official.rst
+.. include:: release_steps/metviewer/update_docs_tagged.rst
+.. include:: release_steps/merge_release_issue.rst
+.. include:: release_steps/create_release_branch.rst
+.. include:: release_steps/push_release_branch.rst
+.. include:: release_steps/create_release_on_github.rst
+.. include:: release_steps/metviewer/create_release_extra.rst
+.. include:: release_steps/finalize_release_on_github_official.rst
```

Comment:
Above this line ".. include:: release_steps/finalize_release_on_github_official.rst" should be .. include:: release_steps/metviewer/update_dtc_website.rst
(We want to ensure that the METviewer releases get added to the DTC website)

Reply:
I am not aware and don't perform this step for METviewer. That is why I removed it from the file